assets/templates/partials/release-alert.tmpl

```diff
@@ -0,0 +1,18 @@
+{{ if .DatasetLandingPage.HasOlderVersions }}
+    <div class="alert-release-banner alert-release-banner__not-latest margin-bottom-sm--2 margin-bottom-md--2 margin-bottom-lg--4 padding-left--1 padding-top--1 padding-bottom--1 padding-right--1">
+        <div class="alert-release-banner__title margin-bottom--0 margin-top--0">
+        This is not the latest data.
+        <a href="{{.DatasetLandingPage.LatestVersionURL}}"> View latest edition </a>
+        </div>
+    </div>
+{{ else }}
+    <div class="alert-release-banner alert-release-banner__latest margin-bottom-sm--2 margin-bottom-md--2 margin-bottom-lg--4 padding-left--1 padding-top--1 padding-bottom--1 padding-right--1">
+        <div class="alert-release-banner__title margin-bottom--0 margin-top--0">
+            This is the latest data.
+            <a href="/datasets/{{.DatasetLandingPage.DatasetID}}/editions/{{.DatasetLandingPage.Edition}}/versions">View previous editions</a>
```

Comment:
Either the text or the href here is wrong. The /versions endpoint will list previous versions of data within this edition. Looking at previous editions would be done through /editions.

In most of our current cases, datasets are produced in time-series, meaning they have only one edition, and we append the new release of data (often monthly) to one contiguous dataset and we call this update a new version

In other cases, datasets are produced in more distinct segments. For instance some projection datasets publish a 2017-projected in Jan 2017 and 2017-revised in Jan 2018 when they have data to be able to reassess and revise the information. 

We don't actually have a way to sort the list of editions, so im not sure if it would be useful to present a user with a list of editions, but from the frontend if we did want to do that it would currently be on this page https://beta.ons.gov.uk/datasets/mid-year-pop-est - if we actually want to send them to previous versions of this edition then:
a) we'd need to check that there _is_ an earlier version (what if this is version 1)
b) the text should probably say previous versions

It's likely this is a requirements clarification for a product manager :)

Reply:
Product manager: "Yeah, should be versions." 

The text has now been changed to.

> "This is not the latest data. View the latest version"


And
"This is the latest data. View previous versions"